bin/resources/GameIndex.yaml

```
@@ -11860,8 +11860,10 @@ SLES-51883:
   name: "Scooby-Doo! Mystery Mayhem"
   region: "PAL-M3"
 SLES-51885:
-  name: "MegaMan X7"
+  name: "Mega Man X7"
   region: "PAL-M5"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering and wrong textures in FMV.
   compat: 5
```

```suggestion
  compat: 5
  gameFixes:
    - SoftwareRendererFMVHack # Flickering and wrong textures in FMV.
```

Comp info should be before the settings themselves.